components/StyledButtonSet.js

```diff
@@ -13,8 +14,11 @@ const StyledButtonItem = styled(StyledButton)`
   border-radius: 0;
   line-height: 1.5;
   flex-grow: 1;
+  border-color: ${themeGet('colors.black.200')};
+
   &:active p {
     color: white;
+    border-color: ${themeGet('colors.red.500')};
```

Comment:
What is the goal you're trying to achieve with this line?

Reply:
ohhhh, I was supposed to remove those 2.. my bad. let me fix it. thanks for the feedback